graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,35 @@
+// import { Context } from '../../@types/helpers'
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+// import { isAdminOrThrow } from '../../helpers/isAdmin'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  //isAdminOrThrow(req) will add after, easier for testing
+  // const authorId = ctx.req.user?.id
+  const { authorId, content, lessonId, name } = args
+  if (!authorId || !content || !name || !lessonId) {
+    throw new Error('Missing parameters')
+  }
+  return await prisma.module.create({
+    data: { authorId, content, lessonId, name }
+  })
+}
+
+export const deleteModule = async (
+  _parent: void,
+  args: MutationDeleteModuleArgs
+) => {
+  // isAdminOrThrow(req)
+  const { id } = args
+  if (!id) throw new Error('Missing parameter')
```

Comment:
question here, if you have marked `id` as required: https://github.com/garageScript/c0d3-app/pull/1354/files#diff-0776d3ad89186de849e067b2a7b9a9701408e7b983a6387c50819581ad6f8f9aR63

Do you still need that check?